docs/generators/spring.md

```diff
@@ -82,6 +83,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |unhandledException|Declare operation methods to throw a generic exception and allow unhandled exceptions (useful for Spring `@ControllerAdvice` directives).| |false|
 |useBeanValidation|Use BeanValidation API annotations| |true|
 |useOptional|Use Optional container for optional parameters| |false|
+|useOptionalInModel|Experimental: Use Optional container for optional model properties| |false|
```

Comment:
I think this should default to true when useOptional is true - but maybe after it graduates from experimental.